app/views/zendesk/intakes/banking_info.html.erb

```diff
@@ -0,0 +1,28 @@
+<% content_for :page_title, "#{@intake.primary_full_name}, ##{zendesk_ticket_id}" %>
+<% content_for :body_class, "admin-details" %>
+<% content_for :card do %>
+  <div class="header">
+    <h1 class="h2">
+      <%= @intake.primary_full_name %>
+    </h1>
+    <p>
+      <%= link_to "##{zendesk_ticket_id}",
+        "https://eitc.zendesk.com/agent/tickets/#{zendesk_ticket_id}" %>
+    </p>
+  </div>
+
+  <div class="info-details">
```

Comment:
Could this class name be made more specific? Maybe `bank-info-details`? I worry that as the CSS gets more complicated this will be hard to keep track of.

Reply:
I was a bit unsure about where this css would go and was trying to make something that could be applied on other detail pages. I am going to leave this since the next story (currently working on) is going to change the styling for these pages and so hopefully things will get clarified there.